spec/features/search_spec.rb

```diff
@@ -56,7 +56,12 @@
       click_link 'Miss Anna S.C. Blake'
       expect(page).to have_content "uarch112-g01650"
 
-      ## Exact title search
+      # IIIF Manifest present on the page
+      expect(page.html).to match(%r{/spotlight/the-anna-s-c-blake-manual-training-school/catalog/1-3/manifest})
```

Comment:
@dunn here's a test that the url for the manifest is included in the page for an item

Reply:
Is it accessible, though?  My theory was that authentication might be required to view it, since I couldn't otherwise explain why you were seeing it on your sandbox and I wasn't on mine.

Reply:
Ah ok! I think this might a different issue when I go to https://geniza-qa.curationexperts.com I'm not getting access to search or anything and when I can't view the show page for an item. I'll check to see if this a configuration problem

Reply:
@dunn @bess  Bess knew what the problem was. The exhibit was not published, which was why you weren't able to access the manifest. I updated the exhibit feature spec so that it tests that non-admin users are able to access the manifest of an item (after the exhibit has been published). It would probably be best to merge this now so I can move on to the next issue. Then if there are more stylistic or syntactical changes make those additional issues.